src/styles/blocks/header.scss

```diff
@@ -0,0 +1,85 @@
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;

```

Comment:
It is forbidden to set sizes and margins for blocks, but for block elements are allowed. The block should stretch to accommodate its children